src/components/Section.tsx

```diff
@@ -0,0 +1,18 @@
+// this comment tells babel to convert jsx to calls to a function called jsx instead of React.createElement
+/** @jsx jsx */
+import { css, jsx } from '@emotion/core';
+import { AccordionRow } from '@guardian/src-accordion';
+import { Tiles } from '@guardian/src-layout';
+import type React from 'react';
+
+export interface SectionProps {
+	sectionName: string;
+}
+
+export const Section: React.FC<SectionProps> = ({ sectionName, children }) => (
+	<div data-testid="section">
+		<AccordionRow label={sectionName}>
```

Comment:
[We recommend](http://theguardian.design/2a1e5182b/p/38c5aa-accordion/b/92b71e) wrapping your `<AccordionRow>` in an `<Accordion>`, if only to add the border to the bottom. Let me know if you have any questions about this 🙂

Reply:
@SiAdcock  I had that originally but had a problem with using `data-testid` with `AccordionRow` and couldn't find another way to count how many `Section` components appeared for the test I put in place. 

Do you know if it's possible to add `data-*` attributes to the Source components?

Reply:
Solution was to place the `<Section>` component inside `<AccordionRow>` and make it a `<div>` with the `data-testid` attribute around the `<Tiles>` component.

Reply:
> Do you know if it's possible to add data-* attributes to the Source components?

Of course! For every component, except Accordion! 😅 

This is a bug, I'll get it fixed. Sorry for the inconvenience.